x-pack/filebeat/docs/inputs/input-aws-s3.asciidoc

```diff
@@ -690,6 +692,32 @@ This option can only be used together with the backup functionality.
 [id="{beatname_lc}-input-{type}-common-options"]
 include::../../../../filebeat/docs/inputs/input-common-options.asciidoc[]
 
+[float]
+==== `ignore_older`
```

Comment:
I think it would be nice to include the format expected, like you did for `start_timestamp`. I had to look at the code to know. Maybe add an example?

Reply:
Good point. I improved this with commit https://github.com/elastic/beats/pull/41817/files/da508e09503e068666e0d509814d9103546eb08e..ce2786ae8561f6bea36f5ade0dd8d3eab44d3cf7